src/app/[locale]/incident/summary/components/IncidentSummaryForm.tsx

```diff
@@ -62,23 +70,36 @@ const IncidentSummaryForm = () => {
         incident_date_start: new Date().toISOString(),
         extra_properties: formState.extra_properties,
       })
-      .then((res) => {
-        if (formState.attachments.length > 0) {
-          const signalId = res.signal_id
-          if (signalId) {
-            formState.attachments.forEach((attachment) => {
-              const formData = new FormData()
-              formData.append('signal_id', signalId)
-              formData.append('file', attachment)
-              postAttachments(signalId, formData)
-            })
+
+      if (formState.attachments.length > 0) {
+        const signalId = res.signal_id
+        if (signalId) {
+          try {
+            await Promise.all(
+              formState.attachments.map(async (attachment) => {
+                const formData = new FormData()
+                formData.append('signal_id', signalId)
+                formData.append('file', attachment)
+                await postAttachments(signalId, formData)
+              })
+            )
+          } catch (e) {
+            // Note: the report does not have fail when one or more of the attachments is not uploaded successfully.
+            console.error('One of the attachments failed while uploading', e)
           }
         }
-      })
-      .then((res) => router.push('/incident/thankyou'))
-      .catch((err) => console.error(err))
+      }
+
+      router.push('/incident/thankyou')
+    } catch (err) {
+      console.error(err)
+      setError(true)
+    } finally {
+      setLoading(false)
+    }
   }
 
+  // @ts-ignore
```

Comment:
Ik snap niet zo goed waarom hier een ```// @ts-ignore``` nodig is? Volgens mij is deze volledig overbodig. Ik krijg in ieder geval geen errors binnen mijn IDE bij het verwijderen van deze.

Reply:
Voorheen gaf het problemen maar nu blijkbaar niet meer, weggehaald 👍